arch/renesas/src/rx65n/rx65n_definitions.h

```diff
@@ -594,6 +594,855 @@
+
+#define RX65N_USB_PIPE0_SIZE                         (256)
+#define RX65N_USB_PIPE_NB_BUFFERS                    (64)
+
+/* Define Synergy static definition.  */
+
+#define RX65N_USB_AVAILABLE_BANDWIDTH                (2304UL)
+
+/* The macro above is used for checking the available bandwidth for periodic
+ * transfers(Isochronous and Interrupt)
+ * Maximum bandwidth is calculated as
+ * {2048byes(2x ISO PIPEs) + 256bytes(4x INT PIPEs)} for high-speed operation
+ */
+ #define RX65N_USB_INIT_DELAY                        (1000)
+#define RX65N_USB_RESET_RETRY                        (1000)
+#define RX65N_USB_RESET_DELAY                        (10)
+#define RX65N_USB_PORT_RESET_RETRY                   (50)
+#define RX65N_USB_FORCE_PORT_RESET_RETRY             (50)
+#define RX65N_USB_FORCE_PORT_RESET_DELAY             (1)
+#define RX65N_USB_CHECK_PORT_RESET_RETRY             (500)
+#define RX65N_USB_PORT_RESET_DELAY                   (300)
+#define RX65N_USB_PORT_RESET_RECOVERY_DELAY          (100)
+
+/* Define Synergy initialization values.  */
+
+#define RX65N_USB_COMMAND_STATUS_RESET               (0)
+#define RX65N_USB_INIT_RESET_DELAY                   (10)
+#define RX65N_USB_MAX_BUF_SIZE                       (64)
+#define RX65N_USB_BUF_BLOCK_SIZE                     (64)
+#define RX65N_USB_MAX_BUF_SIZE_PIPE1_to_2_FS         (256)
+#define RX65N_USB_MAX_BUF_SIZE_PIPE3_to_9_FS         (64)
+#define RX65N_USB_MAX_BUF_SIZE_PIPE1_to_2_HS         (1024)
+#define RX65N_USB_MAX_BUF_SIZE_PIPE3_to_5_HS         (512)
+#define RX65N_USB_MAX_BUF_SIZE_PIPE6_to_9_HS         (64)
+#define RX65N_USB_MAX_BUF_NUM                        (135)
+#define RX65N_USB_PIPE1_BUF_START_NUM                (8)
+
+/* Define Synergy FIFO write completion code.  */
+
+#define RX65N_USB_FIFO_WRITING                       (2)
+#define RX65N_USB_FIFO_WRITE_END                     (3)
+#define RX65N_USB_FIFO_WRITE_SHORT                   (4)
+#define RX65N_USB_FIFO_WRITE_DMA                     (5)
+#define RX65N_USB_FIFO_WRITE_ERROR                   (6)
+
+/* Define Synergy FIFO read completion code.  */
+
+#define RX65N_USB_FIFO_READING                       (2)
+#define RX65N_USB_FIFO_READ_END                      (3)
+#define RX65N_USB_FIFO_READ_SHORT                    (4)
+#define RX65N_USB_FIFO_READ_DMA                      (5)
+#define RX65N_USB_FIFO_READ_ERROR                    (6)
+#define RX65N_USB_FIFO_READ_OVER                     (7)
+#define RX65N_USB_ED_BRDY                            (0x00000001U)
+#define RX65N_USB_ED_NRDY                            (0x00000002U)
+#define RX65N_USB_ED_BEMP                            (0x00000004U)
+#define RX65N_USB_ED_EOFERR                          (0x00000010U)
+#define RX65N_USB_ED_SIGN                            (0x00000020U)
+#define RX65N_USB_ED_SACK                            (0x00000040U)
+#define RX65N_USB_ED_TIMEOUT                         (0x00000080U)
+#define RX65N_USB_LPSTS_SUSPENDM                     (1U<<14)
```

Comment:
Ditto

Reply:
Thank you for the comments.

Alignment errors are corrected in  b3857f3